demisto_sdk/__main__.py

```diff
@@ -734,7 +734,8 @@ def generate_doc(**kwargs):
             return 1
 
     file_type = find_type(kwargs.get('input', ''))
-    if file_type not in [FileType.INTEGRATION, FileType.SCRIPT, FileType.PLAYBOOK, FileType.TEST_SCRIPT]:
+    if file_type not in [FileType.INTEGRATION, file_type.BETA_INTEGRATION, FileType.SCRIPT,
+                         FileType.PLAYBOOK, FileType.TEST_SCRIPT]:
```

Comment:
no reason to have the test script here (we are not generating docs for it)
```suggestion
                         FileType.PLAYBOOK]:
```